binfmt/libelf/libelf_addrenv.c

```diff
@@ -89,27 +90,32 @@ int elf_addrenv_alloc(FAR struct elf_loadinfo_s *loadinfo, size_t textsize,
 
   /* Create an address environment for the new ELF task */
 
-  ret = up_addrenv_create(textsize, datasize, heapsize, &loadinfo->addrenv);
+  ret = up_addrenv_create(textsize, datasize, heapsize,
+                          &loadinfo->addrenv.pgdir);
```

Comment:
add local variable for loadinfo->addrenv.pgdir?

Reply:
Done